### ISSUE TITLE
Fatal error: 'CL/cl.h' file not found

### ISSUE THREAD

**Issue by Vladis432** [fireice-uk/xmr-stak-amd#137]
What version of AMD APP SDK need?
I try to compile in Visual Studio, and get an error Cannot open include file: 'OpenCL/cl.h': No such file or directory.

**Comment by V1pr** [comment]
I had the same issue, but solved it. AMD APP SDK uses 2 locations: one in C:\Users\Username\AMD... and a second in C:\Program files (x86)\AMD. The second one is needed for cmake, not the first!

**Comment by Vladis432** [comment]
I use second one, and it's not working.

What version are you use?

**Comment by V1pr** [comment]
VS is 2017 Communitx (15.4)
AMD APP SDK 3.0
Windows 10 x64
cmake 3.9.4
OpenCL include dir: C:/Program Files (x86)/AMD APP SDK/3.0/include
OpenCL library: C:/Program Files (x86)/AMD APP SDK/3.0/lib/x86_64/OpenCL.lib

Check the windows variable AMDAPPSDKROOT if it's also set to 'C:\Program Files (x86)\AMD APP SDK\3.0'

Lame question: do you've this file? "c:\Program Files (x86)\AMD APP SDK\3.0\include\CL\cl.h"

**Comment by amdvega** [comment]
I'm getting the same error:
  c:\xmr_compile\amd_gpu\gpu.h(3): fatal error C1083: Cannot open include file:
 'CL/cl.h': No such file or directory [C:\xmr_compile\build\xmr-stak-amd.vcxproj]

I confirmed I have these directories:
C:/Program Files (x86)/AMD APP SDK/3.0/include
C:/Program Files (x86)/AMD APP SDK/3.0/lib/x86_64/OpenCL.lib

...and confirmed I have this file:  c:\Program Files (x86)\AMD APP SDK\3.0\include\CL\cl.h"

...I also set AMDAPPSDKROOT to 'C:\Program Files (x86)\AMD APP SDK\3.0' (it wasn't set)

Still not working.  How do I get the build/make process to "see" this missing cl.h file??  The other xmr variants are easy to compile in Windows, this xmr-stak-amd is a pain in the butt!

**Comment by jameswan** [comment]
Has anyone solved this ?

**Comment by psychocrypt** [comment]
yes use our latest version https://github.com/fireice-uk/xmr-stak
This repo here is not maintained anymore.
